frontend/src/features.ts

```diff
@@ -6,6 +6,7 @@ export interface Feature {
 
 export enum FeatureKey {
   V2 = 'v2',
+  V2_ALPHA = 'v2_alpha',
 }
 
 const FEATURE_V2 = {
```

Comment:
I am sorry about the confusion. What I meant is:

Add a comment next to line `V2 = 'v2'` as followed:

```
// Please start using V2_ALPHA instead of V2, because we have switched to V2_ALPHA as V2 feature is enabled by default.
```